app/setup_handlers.go

```diff
@@ -4,13 +4,29 @@ import (
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	dispensationtypes "github.com/Sifchain/sifnode/x/dispensation/types"
 )
 
 const upgradeName = "0.9.12"
 
 func SetupHandlers(app *SifchainApp) {
 	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan types.Plan) {
 		app.Logger().Info("Running upgrade handler for " + upgradeName)
+
+		mintAmount, ok := sdk.NewIntFromString("200000000000000000000000000")
+		if !ok {
+			panic("failed to convert mint amount")
+		}
+		mintCoins := sdk.NewCoins(sdk.NewCoin(app.StakingKeeper.BondDenom(ctx), mintAmount))
+		err := app.BankKeeper.MintCoins(ctx, dispensationtypes.ModuleName, mintCoins)
+		if err != nil {
+			panic(err)
+		}
+		// err = app.BankKeeper.SendCoinsFromModuleToAccount(ctx, dispensationtypes.ModuleName, address, mintCoins) // TODO: get destination address
+		// if err != nil {
```

Comment:
sif1ct2s3t8u2kffjpaekhtngzv6yc4vm97xajqyl3
the destination address, which is the same one we've used for distributions in the past